modules/cnv-storage-wizard-fields-web.adoc

```diff
@@ -10,22 +10,22 @@
 |===
 |Name | Description
 
-|Create Disk
-|Create a new disk for the virtual machine.
+|Source
+|Select a blank disk for the virtual machine or choose from the options available: ISO, URL, Container or Attach Disk. To select an existing disk, choose Attach Disk from a list of available PVCs and attach to the virtual machine. If a new ISO file is uploaded, the filename is displayed in the ISO field.
```

Comment:
It is also possible to attach cloned disk of an existing disk (Attach Cloned disk )